### ISSUE TITLE
2ch.hk [Bug] На последней версии не работает загрузка картинок из буфера обмена.

### ISSUE THREAD

**Issue by NEONZARIN** [SthephanShinkufag/Dollchan-Extension-Tools#1311]
На прошлых версиях загрузка картинок работает. 
![image](https://user-images.githubusercontent.com/16442992/50905772-9c05ee00-1434-11e9-96ae-4e0b97a6fd7f.png)
Хром.
![image](https://user-images.githubusercontent.com/16442992/50905913-f737e080-1434-11e9-8068-1bcdae97b6b6.png)

**Comment by SthephanShinkufag** [comment]
- Как именно вы оставляете картинку? Через `Ctrl+V` в текстовое поле? Где берете картинку (например выделенный кусочек из пеинта)? Подскажите приблизительно.
- Какой скриптменеджер, или у вас расширение?
- Попробуйте отключить все ваши расширения (особенно блокировщики), работает или нет?

**Comment by NEONZARIN** [comment]
Ctrl+V
violentmonkey
Выключил все расширения хрома, работает только скрипт куклы, все равно не работает вставка картинок.

**Comment by SthephanShinkufag** [comment]
Да, бага хрома. Причём и в Violentmonkey, и в тестовом расширении.

**Comment by SthephanShinkufag** [comment]
Исправил.

**Comment by SthephanShinkufag** [comment]
Кстати, не знал про Violentmonkey в хроме. Надо в [вики](https://github.com/SthephanShinkufag/Dollchan-Extension-Tools/wiki/installation) добавить в список вариантов для хрома.

**Comment by NEONZARIN** [comment]
Всё работает, спасибо.
